src/components/Navbar/Navbar.module.css

```diff
@@ -0,0 +1,17 @@
+* {
```

Comment:
If you're going to do this, you should do it in the global styling file, instead of individually for each module of styles